pixi.toml

```diff
@@ -0,0 +1,37 @@
+[project]
+name = "eigenpy"
+version = "3.4.0"
```

Comment:
We'll probably want to bump this in the `release` cmake target too

Reply:
Sure, do you want to do it ?

I'm not sure to merge this PR, I will only do it if I have positive feedback about this tool.

Reply:
I probably won't personaly use this tool, but if we think this tool is going to be used and we want to set it up on our other repos, we'll have to be sure that this version number stays in sync.

Reply:
Fixed in https://github.com/jrl-umi3218/jrl-cmakemodules/pull/684